content/en/api/integrations_webhook/code_snippets/webhooks_update.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+api_key="<DATADOG_API_KEY>"
+app_key="<DATADOG_APPLICATION_KEY>"
+
+curl -v -X PUT \
+-H "DD-API-KEY: ${api_key}" \
+-H "DD-APPLICATION-KEY: ${app_key}" \
+"https://api.datadoghq.com/api/v1/webhooks/configuration/webhooks/<WEBHOOK_NAME>" \
+-d '{
+    "name": "<NEW_WEBHOOK_NAME>",
```

Comment:
Can we show an example that updates something else than the name? `name` is a little bit confusing, because it's also the identifier of the resource.

Reply:
Would you want to have two examples for `update` endpoints then? I think having an example for the more complicated case would be valuable

Reply:
I think it would be good to have one example that updates multiple things

Reply:
This example shows updating both `name` and `encode_as`.